description: type is used so no need to annotate as unused
author: Michael Gilbert <mgilbert@debian.org>

--- a/ui/compositor/test/in_process_context_provider.h
+++ b/ui/compositor/test/in_process_context_provider.h
@@ -105,7 +105,7 @@ class InProcessContextProvider
   std::unique_ptr<gpu::raster::RasterInterface> raster_context_;
   std::unique_ptr<viz::ContextCacheController> cache_controller_;
 
-  const bool support_locking_ ALLOW_UNUSED_TYPE;
+  const bool support_locking_;
   bool bind_tried_ = false;
   gpu::ContextResult bind_result_;
 
--- a/components/viz/test/test_context_provider.h
+++ b/components/viz/test/test_context_provider.h
@@ -174,7 +174,7 @@ class TestContextProvider
   std::unique_ptr<skia_bindings::GrContextForGLES2Interface> gr_context_;
   std::unique_ptr<ContextCacheController> cache_controller_;
   std::unique_ptr<TestSharedImageInterface> shared_image_interface_;
-  const bool support_locking_ ALLOW_UNUSED_TYPE;
+  const bool support_locking_;
   bool bound_ = false;
 
   gpu::GpuFeatureInfo gpu_feature_info_;
